src/main/resources/locales/cy/sscs-daily-hearing-list.json

```diff
@@ -0,0 +1,17 @@
+{
+    "openJusticeStatement": "Mae cyfiawnder agored yn egwyddor hanfodol yn system y llysoedd a’r tribiwnlysoedd. Wrth ystyried defnyddio technoleg ffôn a fideo, bydd y farnwriaeth yn rhoi sylw i egwyddorion cyfiawnder agored. Gall barnwyr benderfynu cynnal gwrandawiad yn breifat os oes angen hynny er mwyn sicrhau'r broses o weinyddu cyfiawnder yn briodol.",
+    "contactMessage": "Bydd partïon a chynrychiolwyr y Tribiwnlys Nawdd Cymdeithasol a Chynnal Plant yn cael gwybod yn uniongyrchol am y trefniadau ar gyfer gwrando achosion o bell. Dylai unrhyw un arall sydd â diddordeb mewn ymuno â’r gwrandawiad o bell gysylltu â Swyddfa’r Tribiwnlys Nawdd Cymdeithasol a Chynnal Plant yn uniongyrchol, cyn dyddiad y gwrandawiad, trwy e-bostio EMAIL fel y gellir gwneud trefniadau. Dylai'r manylion canlynol gael eu cynnwys yn llinell pwnc yr e-bost [OBSERVER/MEDIA] REQUEST",
+    "venue": "Lleoliad",
+    "appealReferenceNumber": "Cyfeirnod Apêl",
```

Comment:
Same like the English one, second word in lower case for all other non-strategic list. Should be consistent.

Reply:
fixed.